MonkeyLoader.Resonite.Integration/Configuration/SettingsDataFeedInjector.cs

```diff
@@ -114,21 +114,21 @@ private static void EnsureColorXTemplate(DataFeedItemMapper mapper)
             if (!_colorXTemplateCleanupDone)
             {
                 Logger.Info(() => "Looking for previously injected colorX templates.");
-                foreach (var mapping in mapper.Mappings.Where((DataFeedItemMapper.ItemMapping mapping) => mapping.MatchingType == typeof(DataFeedValueField<colorX>) && mapping.Template.Target != null && mapping.Template.Target.Slot.Name == InjectedColorXTemplateName).ToList())
+                foreach (var mapping in mapper.Mappings.Where(mapping => mapping.MatchingType == typeof(DataFeedValueField<colorX>) && mapping.Template.Target != null && mapping.Template.Target.Slot.Name == InjectedColorXTemplateName).ToArray())
```

Comment:
You forgot the mapping.Template.Target?.Slot.Name one 😅

Reply:
Fixed